### ISSUE TITLE
Issue with the random calculate for random remeasure

### ISSUE THREAD

**Issue by lbramley** [RobertJohnston/SMART-Questionnaire#13]
Hi,

I believe there's a small issue with the second (random) remeasure group in the SMART Questionnaire. The calculation of the "rand" field in line 5 should be wrapped in a "once()" - so "once(random())" - otherwise if an enumerator goes back to the beginning of the form for any reason, the form recalculates the random number and may change whether they need to do a remeasure or not. This happened to us in a recent field test.

Thanks for providing this really useful tool!

**Comment by RobertJohnston** [comment]
Yes, you are right. Thanks for the note.

I had this in a once() field for the survey in Senegal.  Then we moved to
Liberia to conduct a survey there.  The government had hosted an old
version of formhub which did not have the once() field available. So I am
afraid that I removed it then and never put it back. Will do so now.

If you ever need the official SMART version of the questionnaires, they are
available from http://smartmethodology.org/  ACF Canada.
